frontend/src/app/testResults/TestResultsList.test.tsx

```diff
@@ -552,6 +563,7 @@ describe("TestResultsList", () => {
     });
 
     it("closes the download test results modal after downloading", async () => {
+      // source of "navigation not implemented" error
```

Comment:
tried to fix some of the console errors these tests were generating but couldn't figure them out. Left comments instead in case anyone has oodles of time in the future and wants to fix them